src/js/application.js

```diff
@@ -0,0 +1,288 @@
+            TODAYS_TEMPERATURE_ELEMENT.innerHTML = todayWeather.temperatureInFahrenheit;
+            TODAYS_APPARENT_TEMPERATURE_ELEMENT.innerHTML = ` ${todayWeather.apparentTemperatureInFahrenheit} °`;    
+        }
+               
+        TODAYS_HUMIDITY_ELEMENT.innerHTML = `${todayWeather.humidity} %`;
+        TODAYS_WEATHER_SUMMARY_ELEMENT.innerHTML = todayWeather.summary;
+        TODAYS_WIND_ELEMENT.innerHTML = `${todayWeather.wind} m/s`;
+    }
+    
+    showTemperatureForecast(weather) {
+        const weatherTomorrow = weather.getForecastForDay(1);
+        const weatherSecondDay = weather.getForecastForDay(2);
+        const weatherThirdDay = weather.getForecastForDay(3);
+        const TOMORROWS_TEMPERATURE_ELEMENT = document.querySelector(selectors.TEMPERATURE_TOMORROW);
+        const SECOND_DAYS_TEMPERATURE_ELEMENT = document.querySelector(selectors.TEMPERATURE_SECOND_DAY);
+        const THIRD_DAYS_TEMPERATURE_ELEMENT = document.querySelector(selectors.TEMPERATURE_THIRD_DAY);
+
+        if (this.degreeUnit === 'celsius') {
+            TOMORROWS_TEMPERATURE_ELEMENT.innerHTML = `${weatherTomorrow.temperatureInCelsius}°`;
+            SECOND_DAYS_TEMPERATURE_ELEMENT.innerHTML = `${weatherSecondDay.temperatureInCelsius}°`;
+            THIRD_DAYS_TEMPERATURE_ELEMENT.innerHTML = `${weatherThirdDay.temperatureInCelsius}°`;
+        } else {
+            TOMORROWS_TEMPERATURE_ELEMENT.innerHTML = `${weatherTomorrow.temperatureInFahrenheit}°`;
+            SECOND_DAYS_TEMPERATURE_ELEMENT.innerHTML = `${weatherSecondDay.temperatureInFahrenheit}°`;
+            THIRD_DAYS_TEMPERATURE_ELEMENT.innerHTML = `${weatherThirdDay.temperatureInFahrenheit}°`;
+        }     
+    }
+    
+    showNextDaysOfWeek(weather) {
+        const daysEn = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday'];
+        const daysRu = ['Воскресенье','Понедельник','Вторник','Среда','Четверг','Пятница','Суббота'];
+        const TOMORROW_DAY_OF_WEEK_ELEMENT = document.querySelector(selectors.TOMORROW);
+        const SECOND_DAY_OF_WEEK_ELEMENT = document.querySelector(selectors.SECOND_DAY);
+        const THIRD_DAY_OF_WEEK_ELEMENT = document.querySelector(selectors.THIRD_DAY);
+
+        const weatherTomorrow = weather.getForecastForDay(1);
+        const weatherSecondDay = weather.getForecastForDay(2);
+        const weatherThirdDay = weather.getForecastForDay(3);
+
+        const tomorrowDate = new Date(weatherTomorrow.timestamp * 1000);
+        const secondDayDate = new Date(weatherSecondDay.timestamp * 1000);
+        const thirdDayDate = new Date(weatherThirdDay.timestamp * 1000);
+
+        const days = this.language == 'en' ? daysEn : daysRu;
+        TOMORROW_DAY_OF_WEEK_ELEMENT.innerHTML = days[tomorrowDate.getDay()];
+        SECOND_DAY_OF_WEEK_ELEMENT.innerHTML = days[secondDayDate.getDay()];
+        THIRD_DAY_OF_WEEK_ELEMENT.innerHTML = days[thirdDayDate.getDay()]; 
+    }
+    
+    showWeatherIcons(weather) {
```

Comment:
![image](https://user-images.githubusercontent.com/25482018/115125132-e2ad0000-9fce-11eb-8908-2c6de891f3ea.png)
please test is it possible to use forEach here

Reply:
fixed